packages/foreman/rubygem-nokogiri/rubygem-nokogiri.spec

```diff
@@ -71,8 +71,9 @@ cp -a .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
 mkdir -p %{buildroot}%{gem_extdir_mri}
+mkdir -p %{buildroot}%{gem_extdir_mri}/%{gem_name}
```

Comment:
I was already wondering why no mkdir was needed for it. Technically the line above is no longer needed since it's `-p`.

Reply:
Good call, went ahead and removed the part above.